distribution/docker/transform-log4j-config/build.gradle

```diff
@@ -27,3 +24,23 @@ tasks.named('testingConventions').configure {
   }
 }
 
+String inputFile = "${parent.parent.buildDir}/outputs/default/log4j2.properties"
+String outputFile = "${project.buildDir}/distributions/log4j2.properties"
+String transformSource = "${project.projectDir}/src/main/java/org/elasticsearch/transform/log4j/TransformLog4jConfig.java"
```

Comment:
We should just convert `TransformLog4jConfig` into a proper Gradle task in `buildSrc` rather then rely on `LoggedExec` here. We don't use that anywhere else and it avoids forking another JVM just to execute that logic.

Reply:
In fact, if we do that we can ditch this project entirely and just register the transform task directly in the docker distribution project and get rid of this intermediary project altogether.